src/scrabble.js

```diff
@@ -1,5 +1,62 @@
-function scrabble() {
+aToT = ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't', 'A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']
+dToG = ['d', 'g', 'D', 'G']
+bToP = ['b', 'c', 'm', 'p', 'B', 'C', 'M', 'P']
+fToY = ['f', 'h', 'v', 'w', 'y', 'F', 'H', 'V', 'W', 'Y']
+kay  = ['k', 'K']
+jToX = ['j', 'x', 'J', 'X']
+qToZ = ['q', 'z', 'Q', 'Z']
+
+
+ 
+
+function scrabble(word){
+
+if (typeof word !== 'string'){
+        return 0
+    }    
+    
+total = 0
+
+for (i = 0; i < word.length; i++){
+
+   
+    if (aToT.find(letter => letter === word[i])){
+        total = total + 1
+}   
```

Comment:
This could benefit from touching up the formatting, that closing curly bracket is trying to escape